client/my-sites/upgrades/domain-management/controller.jsx

```diff
@@ -293,8 +291,12 @@ module.exports = {
 		);
 	},
 
-	domainManagementIndex() {
-		page.redirect( '/domains/manage' + ( sites.getSelectedSite() ? ( '/' + sites.getSelectedSite().slug ) : '' ) );
+	domainManagementIndex( pageContext ) {
+		const state = pageContext.store.getState();
+		const siteId = getSelectedSiteId( state );
+		const siteSlug = getSiteSlug( state, siteId );
```

Comment:
Yes, I didn't know it exists :) I followed a similar use case in the same file. I'll refactor both.